src/protocolsupport/injector/pe/PEProxyServerInfoHandler.java

```diff
@@ -53,7 +53,7 @@ public String getServerInfo(Channel channel) {
 			return String.join(";",
 				"MCPE",
 				ping.getMotd().toLegacyText().replace(";", ":"),
-				String.valueOf(ping.getVersion().getName()), "1.12.13",
+				String.valueOf(ping.getVersion().getName()), "1.7.0",
```

Comment:
Can you do this like in PS main? https://github.com/ProtocolSupport/ProtocolSupport/blob/90c738615ddb9c898735f3ae9360baad031c9e4b/src/protocolsupport/zplatform/impl/pe/PEProxyServerInfoHandler.java#L64